client/app/account/billing/autoRenew/billing-autoRenew.html

```diff
@@ -4,57 +4,34 @@
         <h1 data-translate="autorenew_title"></h1>
     </div>
 
-    <div class="tab-content">
+    <div class="text-center" data-ng-if="initLoading">
+        <oui-spinner data-size="l"></oui-spinner>
+    </div>
+
+    <div class="tab-content" data-ng-if="!initLoading">
 
         <div data-ovh-alert></div>
 
         <div class="alert alert-danger"
              role="alert"
-             data-ng-if="!automaticRenewV2Mean.loading && automaticRenewV2Mean.available && (!automaticRenewV2Mean.allowed || (automaticRenewV2Mean.allowed && !nicRenew.active))">
+             data-ng-if="!automaticRenewV2Mean.loading && automaticRenewV2Mean.available && (!automaticRenewV2Mean.allowed || (automaticRenewV2Mean.allowed && !nicRenew.active)) && services.data.userMustApproveAutoRenew">
             <button type="button"
                     class="close"
                     data-ng-click="automaticRenewV2Mean.noMeanMessageClose()"
                     title="{{ 'browser_alert_close' | translate }}"
                     data-dismiss="alert">
             </button>
-            <ul>
-                <li data-ng-if="!automaticRenewV2Mean.allowed" data-translate="autorenew_service_renew_requires_mean_and_date"></li>
-                <li data-ng-if="automaticRenewV2Mean.allowed && !nicRenew.active">
-                    <span data-translate="autorenew_service_activate_alert"></span>
-                    <button class="btn btn-default"
-                            type="button"
-                            data-ng-click="setAction('active', { nicRenew: nicRenew }, 'autoRenew')"
-                            data-translate="autorenew_service_active_title">
-                    </button>
-                </li>
-            </ul>
-        </div>
-
-        <div class="alert alert-info"
-             role="alert"
-             data-ng-if="automaticRenewV2Mean.available && !hideAutorenewMassMigrationAlert">
-            <button type="button"
-                    class="close"
-                    data-ng-click="hideAutorenewMassMigrationAlert = true"
-                    data-dismiss="alert">
-            </button>
-            <span data-translate="autorenew_service_renew_mass_migration_soon"></span>
+            <p data-ng-if="!automaticRenewV2Mean.allowed" data-translate="autorenew_service_renew_requires_mean_and_date"></p>
+            <p data-ng-if="automaticRenewV2Mean.allowed && !nicRenew.active">
+                <span data-translate="autorenew_service_activate_alert"></span>
+                <button class="btn btn-default"
+                        type="button"
+                        data-ng-click="setAction('active', { nicRenew: nicRenew }, 'autoRenew')"
+                        data-translate="autorenew_service_active_title">
+                </button>
+            </p>
         </div>
 
-        <p class="alert alert-info"
```

Comment:
Removed this `alert` as it's not longer relevant (no guide matches this part of the Manager)